week-2/L-array-methods-2/exercise2.js

```diff
@@ -7,7 +7,11 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  var str = split (separator) 'h');
```

Comment:
I'm a bit confused here. What's this doing? It's quite a tricky problem. A good idea is to search for the functions it suggests to use. I just did and I found [this page](https://www.w3schools.com/jsref/jsref_split.asp), which also has a useful tip:

> Tip: If an empty string ("") is used as the separator, the string is split between each character.

Playing with that and the `join` function in node:

```
> "daniel".split("");
[ 'd', 'a', 'n', 'i', 'e', 'l' ]
> "daniel".split("").join();
'd,a,n,i,e,l'
> "daniel".split("").join("");
'daniel'
```
So if you can find out how to convert a string to upper case, you can split the string into an array  of characters, upper case the first element of the array, then join them back together.